packages/components/src/components/infotext/infotext.lite.tsx

```diff
@@ -0,0 +1,62 @@
+import { onMount, Show, useMetadata, useStore } from '@builder.io/mitosis';
+import { DBIcon } from '../icon';
+import { DBInfotextState, DBInfotextProps } from './model';
+
+useMetadata({
+	isAttachedToShadowDom: true,
+	component: {
+		includeIcon: false,
+		properties: []
+	}
+});
+
+export default function DBInfotext(props: DBInfotextProps) {
+	const state = useStore<DBInfotextState>({
+		getIcon: (
+			icon?: string,
+			variant?:
+				| 'adaptive'
+				| 'critical'
+				| 'information'
+				| 'warning'
+				| 'success'
+		) => {
+			if (icon) return icon;
+			switch (variant) {
+				case 'critical':
+					return 'error';
+				case 'warning':
+					return 'error-triangle';
+				case 'success':
+					return 'check-circle';
+				default:
+					return 'info';
+			}
+		}
+	});
+
+	onMount(() => {
+		if (props.stylePath) {
+			state.stylePath = props.stylePath;
+		}
+	});
+
+	return (
+		<div
```

Comment:
Is this really supposed to be a block level element, or isn't it inline?

```suggestion
		<span
```

Reply:
@mfranzke I had the same question when implementing this for RI UI. In case of RI UI (for example) we need this to be a block element. We could discuss if this could be a data attribute as well?